backend/generate-typeshare.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env sh
 
 SCRIPT_DIR=$(dirname -- "$0")
+
+# `typeshare` must be available; use `cargo install typeshare-cli` or equivalent
```

Comment:
Perhaps we could instead check if typeshare-cli is available and if not, echo a message asking the user to install it. What do you think?